### PR TITLE
Fix/adjust ingress api version

### PR DESCRIPTION
**Pull request by davetromp**
The new version requires a different structure for the rules in the yaml files.

### CODE REVIEW

File: charts/layer0_describo/templates/ingress.yaml [hunk 1]
```diff
@@ -15,10 +15,13 @@ spec:
       secretName: {{ include "common.tlsSecretName" . }}
   {{- end }}
   rules:
-    - host: {{ .Values.global.describo.domain }}
+    - host: {{ .Values.global.rds.domain }}
```
Comment (juriroemer) [inline charts/layer0_describo/templates/ingress.yaml]: I think this is a copy/paste mistake?

Comment (davetromp) [reply charts/layer0_describo/templates/ingress.yaml]: Yes, it is. Let me fix this fix :-)

File: charts/layer0_describo/templates/ingress.yaml [hunk 2]
```diff
@@ -15,10 +15,13 @@ spec:
...
         paths:
           - path: {{ .Values.ingress.path }}
+            pathType: Prefix
             backend:
-              serviceName: {{ $fullName }}
-              servicePort: http
+              service:
+                name: {{ $fullName }}
+                port:
+                  number: 80
```
Comment (Deaddy) [inline charts/layer0_describo/templates/ingress.yaml]: Is it possible to refer to the port by name still, i.e. `name: http`?

Comment (davetromp) [reply charts/layer0_describo/templates/ingress.yaml]: Not that I can tell from the docs: https://kubernetes.io/docs/concepts/services-networking/ingress/

File: charts/layer0_describo/templates/ingress.yaml (hunk 1 above)
Comment (davetromp) [reply charts/layer0_describo/templates/ingress.yaml]: OK fixed it

File: charts/layer0_describo/templates/ingress.yaml (hunk 2 above)
Comment (Deaddy) [reply charts/layer0_describo/templates/ingress.yaml]: As discussed in gitter and just for future reference/reviewers, it should still work according to API docs:
https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.22/#servicebackendport-v1-networking-k8s-io

Comment (davetromp) [reply charts/layer0_describo/templates/ingress.yaml]: I tested it and  yes it does also work with name: http

**Review by Deaddy — APPROVED** [review]
lgtm

**Pull request merged**